modules/machineset-yaml-rhv.adoc

```diff
@@ -0,0 +1,108 @@
+// Module included in the following assemblies:
+//
+// * machine_management/creating-infrastructure-machinesets.adoc
+// * machine_management/creating_machinesets/creating-machineset-rhv.adoc
+
+[id="machineset-yaml-rhv_{context}"]
+=  Sample YAML for a machine set custom resource on {rh-virtualization}
+
+This sample YAML defines a machine set that runs on {rh-virtualization} and creates nodes that are labeled with `node-role.kubernetes.io/<node_role>: ""`.
+
+In this sample, `<infrastructure_id>` is the infrastructure ID label that is based on the cluster ID that you set when you provisioned the cluster, and `<role>` is the node label to add.
+
+[source,yaml,subs="+quotes"]
+----
+apiVersion: machine.openshift.io/v1beta1
+kind: MachineSet
+metadata:
+  labels:
+    machine.openshift.io/cluster-api-cluster: <infrastructure_id> <1>
+    machine.openshift.io/cluster-api-machine-role: <role> <2>
+    machine.openshift.io/cluster-api-machine-type: <role> <2>
+  name: <infrastructure_id>-<role> <3>
+  namespace: openshift-machine-api
+spec:
+  replicas: <number_of_replicas> <4>
+  Selector: <5>
+    matchLabels:
+      machine.openshift.io/cluster-api-cluster: <infrastructure_id> <1>
+      machine.openshift.io/cluster-api-machineset: <infrastructure_id>-<role> <3>
+  template:
+    metadata:
+      labels:
+        machine.openshift.io/cluster-api-cluster: <infrastructure_id> <1>
+        machine.openshift.io/cluster-api-machine-role: <role> <2>
+        machine.openshift.io/cluster-api-machine-type: <role> <2>
+        machine.openshift.io/cluster-api-machineset: <infrastructure_id>-<role> <3>
+    spec:
+      metadata:
+        labels:
+          node-role.kubernetes.io/<role>: "" <2>
+      providerSpec:
+        value:
+          apiVersion: ovirtproviderconfig.machine.openshift.io/v1beta1
+          cluster_id: <ovirt_cluster_id> <6>
+          template_name: <ovirt_template_name> <7>
+          instance_type_id: <instance_type_id> <8>
+          cpu: <9>
+            sockets: <number_of_sockets> <10>
+            cores: <number_of_cores> <11>
+            threads: <number_of_threads> <12>
+          memory_mb: <memory_size> <13>
+          os_disk: <14>
+            size_gb: <disk_size> <15>
+          network_interfaces: <16>
+            vnic_profile_id:  <vnic_profile_id> <17>
+          credentialsSecret:
+            name: ovirt-credentials <18>
+          kind: OvirtMachineProviderSpec
+          type: <workload_type> <19>
+          userDataSecret:
+            name: worker-user-data
+----
+<1> Specify the infrastructure ID that is based on the cluster ID that you set when you provisioned the cluster. If you have the OpenShift CLI (`oc`) installed, you can obtain the infrastructure ID by running the following command:
++
+[source,terminal]
+----
+$ oc get -o jsonpath='{.status.infrastructureName}{"\n"}' infrastructure cluster
+----
+
+<2> Specify the node label to add.
```

Comment:
This says node label, but the annotation seems to want a _role_. Are these the same?